src/utils/components/Consoles/components/vnc-console/utils/util.ts

```diff
@@ -1,2 +1,11 @@
 // keyboard keys that need to press&hold the shiftKey
-export const isShiftKeyRequired = (char: string): boolean => '~!@#$%^&*()_+}{|":?><'.includes(char);
+export const isShiftKeyRequired = (char: string): boolean =>
+  !!char?.match(/[A-Z~!@#$%^&*()_+}{|\":?><]/);
+
+// Line Feed/ New Line
+export const LF_NL = 10;
+// horizontal tab
+export const HT = 9;
+
```

Comment:
Please avoid acronyms

Reply:
A rule of thumb, if it needs a comment to explain, it's not a good name/code block